Assets/Scripts/Nathan/PauseMusic.cs

```diff
@@ -0,0 +1,26 @@
+using UnityEngine;
+
+public class PauseMusic : MonoBehaviour
+{
+    // This script also keeps track of pause, so that I knows when it changed.
+    private bool pause;
+
+    // Start is called before the first frame update
+    void Start()
```

Comment:
Remove empty method

Reply:
It will be used later.

Reply:
We can leave it there its not hurting anything.